### PR TITLE
[ingester] add OTel events

### PR DESCRIPTION
**Pull request by numachen**

### CODE REVIEW

File: server/querier/db_descriptions/clickhouse/tag/flow_log/l7_flow_log.en [hunk 1]
```diff
@@ -72,6 +72,7 @@ response_status           , Response Status               ,
 response_code             , Response Code                 ,
 response_exception        , Response Exception            ,
 response_result           , Response Result               ,
+events , Events ,
```
Comment (sharang) [inline server/querier/db_descriptions/clickhouse/tag/flow_log/l7_flow_log.en:75]: 对齐到上一行

**Comment by sharang** [comment]
> [CLA assistant check](https://cla-assistant.io/deepflowio/deepflow?pullRequest=5575) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/deepflowio/deepflow?pullRequest=5575) before we can accept your contribution.You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/deepflowio/deepflow?pullRequest=5575) it.

@numachen ，需要处理一下 CLA

File: server/querier/db_descriptions/clickhouse/tag/flow_log/l7_flow_log.en (hunk 1 above)
Comment (numachen) [reply server/querier/db_descriptions/clickhouse/tag/flow_log/l7_flow_log.en]: ok

File: server/ingester/common/const.go [hunk 2]
```diff
@@ -17,6 +17,6 @@
 package common
 
 const (
-	CK_VERSION             = "v6.5.0.0" // 用于表示clickhouse的表版本号
+	CK_VERSION             = "v6.4.4.2" // 用于表示clickhouse的表版本号
```
Comment (lzf575) [inline server/ingester/common/const.go:20]: version set  "v6.5.0.1"

**Review by lzf575 — APPROVED** [review]

File: server/ingester/common/const.go (hunk 2 above)
Comment (lzf575) [reply server/ingester/common/const.go:20]: 6.4 中就需要 设置为 当前6.4的version  + 1即可。

File: server/ingester/flow_log/log_data/otel.go [hunk 3]
```diff
@@ -328,6 +328,10 @@ func (h *L7FlowLog) FillOTel(l *v1.Span, resAttributes []*v11.KeyValue, platform
 		h.ResponseDuration = uint64(h.L7Base.EndTime - h.L7Base.StartTime)
 	}
 
+	if eventsJSON, err := json.Marshal(l.Events); err == nil {
```
Comment (lzf575) [inline server/ingester/flow_log/log_data/otel.go:331]: 编译错误，需要加下 import 下 "encoding/json"

**Review by lzf575 — APPROVED** [review]

File: server/ingester/flow_log/log_data/otel.go (hunk 3 above)
Comment (sharang) [reply server/ingester/flow_log/log_data/otel.go:331]: @lzf575 有更高性能的 json 库选择吗？我们其他的地方也是用的这个 json 库？

Comment (lzf575) [reply server/ingester/flow_log/log_data/otel.go:331]: > @lzf575 有更高性能的 json 库选择吗？我们其他的地方也是用的这个 json 库？

@sharang 当前api组使用的高性能json是 https://github.com/goccy/go-json，可以切换为这个，是否需要再验证下功能。

Comment (lzf575) [reply server/ingester/flow_log/log_data/otel.go:331]: 重新提了个 v64 的pr  https://github.com/deepflowio/deepflow/pull/5638    @sharang @numachen

**Comment by lzf575** [comment]
新的pr  https://github.com/deepflowio/deepflow/pull/5638

**Pull request closed without merging**